librz/bin/format/pe/pe_info.c

```diff
@@ -137,6 +138,7 @@ char *PE_(rz_bin_pe_get_arch)(RzBinPEObj *bin) {
 		break;
 	case PE_IMAGE_FILE_MACHINE_POWERPC:
 	case PE_IMAGE_FILE_MACHINE_POWERPCFP:
+	case PE_IMAGE_FILE_MACHINE_POWERPC_BE:
```

Comment:
does BE stand for big endian? if yes, then should we expect different parsing for this variation?

Reply:
Yes, it's Big Endian. This should be noticed at least in the disassembler. (PPC Big Endian arch)

Reply:
then maybe you want to set the info->big_endian boolean to true

Reply:
I am not sure.